test/Worker.Extensions.Tests/Worker.Extensions.Tests.csproj

```diff
@@ -19,6 +19,13 @@
             <PrivateAssets>all</PrivateAssets>
```

Comment:
Should we consider moving any new project/code in that structure already? @jviau 

  - extensions
       - Worker.Extensions.Http.AspNetCore
           - src
                   - Worker.Extensions.Http.AspNetCore
                   - Worker.Extensions.Http.AspNetCore.Analyzers.
           - tests
                   - Worker.Extensions.Http.AspNetCore.Analyzers.Tests

Reply:
Not sure how early we are planning to shift to new restructuring of the folders. But meanwhile, I am thinking that it can look bit awkward to have only 1 category of tests inside `Extensions` folder and all remaining tests inside separate tests projects.

Reply:
Yeah, the engineering work (when I get to it) is planned to restructure tests. But until then feel free to put tests wherever you feel they fit best. If you want you could add one similar to this: https://github.com/Azure/azure-functions-dotnet-worker/tree/main/test/Worker.Extensions.Rpc.Tests

Reply:
Done